opentelemetry/src/lib.rs

```diff
@@ -96,6 +96,13 @@
 //! * `metrics`: Includes the unstable metrics API.
 //! * `logs`: Includes the logs bridge API.
 //!
+//! The following feature flags provides additional configuration for `logs`:
+//! * `logs_level_enabled`: Allow users to control the log level
+//!
+//! The following feature flags enables unstable APIs, those APIs are subject to changes without a major version bump,
```

Comment:
one caveat here is -- it may give the impression that the rest of the APIs are stable.. May not be any real issue, but just called out as I found it potentially misleading...

Reply:
True. I will remove it for now